### PR TITLE
stub for acceptance suite

### PR DESCRIPTION
**Pull request by turadg**
refs: #8961

## Description

Start of #8961, so far just moving the core-eval tests out of upgrade-next so they don't have to be copied forward each time ([comment](https://github.com/Agoric/agoric-sdk/issues/8961#issuecomment-2057771664)).

### CODE REVIEW

File: a3p-integration/proposals/a:upgrade-next/core-eval-test-submission/send-script.tjs
Comment (mhofman) [inline a3p-integration/proposals/a:upgrade-next/core-eval-test-submission/send-script.tjs:1]: Did this PR forget to re-add the `core-eval-test-submission` files in the other proposal?

Comment (turadg) [reply a3p-integration/proposals/a:upgrade-next/core-eval-test-submission/send-script.tjs:1]: it was gitignored by default. a:upgrade-next had it force added. I've reset the gitignore for this dir and force pushed

File: a3p-integration/proposals/z:acceptance/README.md
Comment (gibson042) [inline a3p-integration/proposals/z:acceptance/README.md]:
```suggestion
Its `type` is `/cosmos.params.v1beta1.ParameterChangeProposal` because that is
```

Comment (turadg) [reply a3p-integration/proposals/z:acceptance/README.md]: Other than that do you Approve? It would save time to say that you approve it conditional on some requests and rely on the author to make them before merging.

**Review by dckc — APPROVED** [review]
this looks like a useful start

**Comment by dckc** [comment]
please don't merge until #9238 lands

**Pull request merged**
